src/tools/cgsnapshot.c

```diff
@@ -767,20 +767,24 @@ int main(int argc, char *argv[])
 		output_f = stdout;
 
 	/* blacklkist */
-	if (flags & FL_BLACK) {
+	if (flags & FL_BLACK)
```

Comment:
nitpick - I thought the kernel syntax recommendation in a case like this was to keep the braces.  Am I old and out of touch? :)

https://www.kernel.org/doc/html/v4.10/process/coding-style.html

Reply:
@drakenclimber Thanks for the review, it's a long patch set. I am wondering why the `checkpatch.pl` did not flag it. I will try to figure out what went wrong. Agreed, will fix it.